client/src/pages/Login.js

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import { makeStyles } from "@material-ui/styles";
+import Grid from "@material-ui/core/Grid";
+import TextField from "@material-ui/core/TextField";
+import Typography from "@material-ui/core";
+import withStyles from "@material-ui/core/styles";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import Image from "../assets/bg-img.png";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    height: "100vh",
+    fontFamily: "Open Sans"
+  },
+  login: {
+    marginTop: "20px",
+    display: "flex",
+    direction: "row",
+    justify: "space-around",
+    alignItems: "center",
+    float: "right"
+  },
+  createButton: {
+    background: "white",
+    fontFamily: "Open Sans",
+    fontWeight: "600",
+    color: "#3A8DFF",
+    padding: "20px 70px 20px 70px",
+    marginLeft: "20px"
+  },
+  loginButton: {
+    backgroundColor: "#3A8DFF",
+    fontFamily: "Open Sans",
+    fontWeight: "600",
+    padding: "20px 70px 20px 70px",
+    marginTop: "50px"
+  },
+  speechIcon: {
+    color: "white",
+    textAlign: "center"
+  },
+  motto: {
+    display: "inline-block",
+    color: "white",
+    textAlign: "center",
+    marginTop: "40%",
+    fontSize: "200%"
+  },
+  image: {
+    backgroundImage: `linear-gradient(to bottom, rgba(58, 141, 255, 0.85), rgba(134, 185, 255, 0)), url(${Image})`,
+    backgroundRepeat: "no-repeat",
+    backgroundHeight: "100%",
+    backgroundPosition: "center",
+    backgroundSize: "100% 100%",
+    padding: "50px",
+    height: "100vh",
+  },
+  form: {
+    textAlign: "center",
+    display: "inline-block",
+    marginLeft: "50%",
+    marginTop: "20%"
+  }
+}));
+
+export default function Signup() {
+  const classes = useStyles();
+
+  return (
+    <Grid container component="main" className={classes.root}>
+      <CssBaseline />
+      <Grid item container xs={false} sm={4} md={4}>
+        <div id="signUpContent" className={classes.image}>
+          <div className={classes.motto}>
+            <div className={classes.speechIcon}>
+              <i class="far fa-comment-dots fa-5x" />
+            </div>
+            <p>Converse with anyone with any language.</p>
+          </div>
+        </div>
+      </Grid>
+      <Grid item xs={6} sm={8} md={5}>
+        <div className={classes.login}>
+          <div>
+            <p>Don't have an account?</p>
+          </div>
+          <Button
```

Comment:
I noticed you didn't implement this yet, you can do that pretty easily with a Link component from react-router-dom and style it as a button